<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>735</width>
    <height>433</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>735</width>
    <height>433</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>735</width>
    <height>433</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="MyWidget" name="Graphwidget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>571</width>
      <height>411</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="NowState">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>10</y>
      <width>141</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>实时状态</string>
    </property>
    <widget class="QLabel" name="NowSpeedlabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>转速</string>
     </property>
    </widget>
    <widget class="QLabel" name="NowControllabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>控制</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="NowSpeedlineEdit">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="NowControllineEdit">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>50</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>90</y>
      <width>141</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>控制参数</string>
    </property>
    <widget class="QLineEdit" name="KplineEdit">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>0.73</string>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="Kplabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Kp</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="KilineEdit">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>50</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>0.04</string>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="Kilabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Ki</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="KdlineEdit">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>70</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>0.001</string>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="Kdlabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Kd</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="TargetSpeedlineEdit">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>90</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>2700</string>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="TargetSpeedlabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>转速</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="StartTestpushButton">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>340</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>启动测试</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="SerialPortgroupBox">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>210</y>
      <width>141</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>串口设置</string>
    </property>
    <widget class="QComboBox" name="SerialPortcomboBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="FreshSerialpushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>121</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>刷新列表</string>
     </property>
    </widget>
    <widget class="QPushButton" name="OpenSerialpushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>121</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>打开串口</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="StopTestpushButton">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>370</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>停止测试</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SaveDatapushButton">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>400</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>保存数据</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyWidget</class>
   <extends>QWidget</extends>
   <header>mywidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
